src/Symfony/Component/DependencyInjection/ContainerInterface.php

```diff
@@ -37,7 +37,9 @@ public function set(string $id, ?object $service): void;
      *
      * @param B $invalidBehavior
      *
-     * @psalm-return (B is self::EXCEPTION_ON_INVALID_REFERENCE|self::RUNTIME_EXCEPTION_ON_INVALID_REFERENCE ? object : object|null)
+     * @psalm-return ($id is class-string<A> ? (B is self::RUNTIME_EXCEPTION_ON_INVALID_REFERENCE|self::EXCEPTION_ON_INVALID_REFERENCE ? A : A|null) : (B is self::RUNTIME_EXCEPTION_ON_INVALID_REFERENCE|self::EXCEPTION_ON_INVALID_REFERENCE ? object : object|null))
```

Comment:
isn't this missing a `@template A` to even be valid ?

Reply:
Looking at the CI errors, there is definitely something missing:
> Error: src/Symfony/Bridge/Doctrine/Middleware/IdleConnection/Listener.php:41:21: UndefinedDocblockClass: Docblock-defined class, interface or enum named Symfony\Component\DependencyInjection\A does not exist (see https://psalm.dev/200)